summary/summary.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+from lightning import Plugin, Millisatoshi
+import json
+import requests
+import threading
+import time
+
+plugin = Plugin(autopatch=True)
+
+
+class PriceThread(threading.Thread):
+    def run(self):
+        try:
+            r = requests.get('https://apiv2.bitcoinaverage.com/convert/global'
+                             '?from=BTC&to={}&amount=1'.format(plugin.currency))
+            plugin.fiat_per_btc = json.loads(r.content)['price']
+        except Exception:
+            pass
+        # Six hours is more than often enough for polling
+        time.sleep(6*3600)
```

Comment:
Wait, we don't get direct integration with the ICE datafeed? I'm disappointed :wink:

Reply:
We should ask, yes!